app/bg/background.js

```diff
@@ -127,6 +127,37 @@ let onCommandListener = function(cmd) {
 } //onCommandListener()
 chrome.commands.onCommand.addListener(onCommandListener);
 
+// When a window is created
+chrome.windows.onCreated.addListener(function(window){
+  // Leave TabFern window alone...
+  if (window.id === me.viewWindowID) return;
+  
+  if (S.getBool(S.FIXED_WINDOW_SIZE) && window.type === 'normal') {
+    let windowSize = S.getString(S.S_WINDOW_SIZE);
+    [windowWidth, windowHeight] = windowSize.split('x').map((value) => parseInt(value));
```

Comment:
Why a string of `WxH` instead of separate width and height fields?  (And the same throughout.)

E.g., what if the user types `800 x 600` (with spaces around the `x`) or `800` (forgetting the `x H`)?  I personally would prefer two separate numeric fields.

However, I do see an advantage of a single field.  With a single field, it's either set or it's not.  Perhaps we could change settings.js to improve the validation.

What are your thoughts?